api/v1/tetragon/tetragon.proto

```diff
@@ -277,3 +277,9 @@ message HealthStatus {
 message GetHealthStatusResponse {
 	repeated HealthStatus health_status = 1;
 }
+
```

Comment:
How about adding a short comment about what this event is exactly?

Reply:
yes, will add